plugins/panels/similarity_search/run_manager.py

```
@@ -40,7 +40,7 @@ class RunManager:
     _OPID_PREFIX = "opid:"
 
     # Fields too large for listing — only needed by get_run / apply_run
-    _HEAVY_FIELDS = {"result_ids", "result_view", "source_view"}
+    _HEAVY_FIELDS = {"result_ids", "result_view"}
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)